examples/rust-examples/sort-numbers/src/main.rs

```diff
@@ -17,7 +17,7 @@
 use std::fs;
 use anyhow;
 
-const INPUT_FILENAME: &'static str = "/input/unsorted_numbers.txt";
+const INPUT_FILENAME: &'static str = "/output/unsorted_numbers.txt";
```

Comment:
are we sure this is correct?

Reply:
Yes. This program will read another wasm program's result. The test case here is to execute two wasm binaries, `random_u32_list.wasm` and `sort_number.wasm`. The former will general a list of u32, and the latter, which is this program, read the result and sort it.

The test case is `basic_pipeline` in `server_tests.rs`.